src/resources/views/dashboard/surveys/appendices/surveyAppendixS.blade.php

```diff
@@ -1,81 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
 
-@section('surveyAppendixS')
-	<div class="survey-title">
-        <h2>Appendix S: Work Productivity And Activity Impairment Questionnaire General Health (WPAI-GH)</h2>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- CSS ORDER MATTERS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/dashboard/index.css" />
+    <link rel="stylesheet" href="/css/dashboard/sidebar.css" />
+	<link rel="stylesheet" href="/css/dashboard/surveyrightbar.css" />
+
+	<script src="https://kit.fontawesome.com/555936ed9c.js" crossorigin="anonymous"></script>
+    <title>Appendix S: Work Productivity And Activity Impairment Questionnaire General Health (WPAI-GH)</title>
+
+</head>
+
+<body>
+<?php
+    // Enter host name, database username, password, and database name.
+    // connect to database.
+    $con = mysqli_connect("localhost","root","","DatabaseName");
+    // Check connection
+    if (mysqli_connect_errno()){
+        echo "Failed to connect to MySQL: " . mysqli_connect_error();
+    }
+
+    // When form submitted, insert values into the database.
+    if (isset($_POST['submit'])) {
+        //Put survey answers into variables.
+		$answer1 = $_POST['question'];
+		$answer2 = $_POST['question2'];
+		$answer3 = $_POST['question3'];
+		$answer4 = $_POST['question4'];
+		$answer5 = $_POST['question5'];
+		$answer6 = $_POST['question6'];
```

Comment:
after a post request this file goes to the routes. Use the routes to send the data to a controller. The controller will then take in the different variables through a $request object that is in the function call (function store(Request $request) {} for example).